src/main/java/fworks/da/PostgresAccessManager.java

```diff
@@ -138,4 +138,52 @@ public boolean saveNewUserInfo(URegisterDsRequestModel requestModel) {
         }
     }
 
+    // Generate random alphanumeric ID
+    public static String randomIdGenerator(){
```

Comment:
This looks great and will be very useful when implementing all other save methods for entities (where id generation is required).